extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -61,6 +64,10 @@ class AmpStickyAd extends AMP.BaseElement {
         'amp-sticky-ad must have a single amp-ad child');
 
     this.ad_ = children[0];
+    const adType = this.element.getAttribute('type');
+    const config = adConfig[this.ad_.getAttribute('type')];
+    user().assert(config, `Type "${adType}" is not supported in amp-ad`);
```

Comment:
we get the chance to assert this ahead. Or do you think we should  make the code clean and only check once in amp-ad `#layoutCallback`